pybv/io.py

```diff
@@ -64,10 +64,12 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
         If float, the same resolution is applied to all channels.
         If ndarray with n_channels elements, each channel is scaled with
         its own corresponding resolution from the ndarray.
-        Note that if `fmt` is 'binary_int16', it is recommended to set
-        `resolution` to '0.1' (default) and `unit` to 'µV' (default).
-        For `fmt` 'binary_float32', the resolution should usually be kept
-        at its default value.
+        Note that `resolution` is applied on top of the default resolution
+        that a data format (see `fmt`) has; that is, 0 floating point
+        resolution for binary_int16, and 1e-6 floating point resolution for
+        binary_float32. For example, writing binary_int16 data in µV with
```

Comment:
This is still not clear to me. What does "0 floating point resolution for binary_int16" mean?

Reply:
What I want to express with this, is that if you write, say,  `1.234` to int16, you will end up with `1`, the floating point resolution is 0. Does that make sense?

Reply:
@cbrnr @hoechenberger for this param docstr your input would be highly valued, so that we can make it straight forward for users

Reply:
Ah I see! I think it is confusing to talk about a "floating point resolution" when you want to save an int. I think the example already makes it pretty clear, so what about deleting the sentence starting with "Note that..."?

Reply:
I think that would be okay, and less confusing --> but it also removes some information. For example `np.finfo(np.float32)` tells me that the resolution of float32 on my system is `1e-6` --> now if I apply the pybv resolution of `0.1`, I'll actually have a `1e-7` resolution, not a `0.1` resolution - or am I overthinking this?

Reply:
Hm. This is really pretty complicated. The resolution of float32 will always be 1e-6, but the resolution parameter effectively scales the data so you can shift the unit and therefore effectively change the resolution. I really wonder why they decided to include such a parameter, it's probably only necessary if you store integer data. Maybe we should focus on this use case in our explanations?